src/usr/console/uart.C

```diff
@@ -117,7 +117,7 @@ namespace CONSOLE
             // Wait for transmit FIFO to have space.
             {
                 const uint64_t DELAY_NS = 100;
-                const uint64_t DELAY_LOOPS = 10000;
+                const uint64_t DELAY_LOOPS = 100000000;
```

Comment:
Could you add an explanation for this in the commit message please?

Reply:
I've got no idea why this was done. The commit message from 2014 was as follows:

> Changes for Habanero bringup, uart delay and centaur/vddr 

Note that every openpower machine has carried this change. It was placed in the op-build tree instead of committed to the hostboot repository for unknown reasons.

Reply:
I have no explanation beyond wishful thinking on why it remained as a patch for so long.